src/raftstore/store/peer.rs

```diff
@@ -966,6 +1047,19 @@ impl Peer {
                 // apprently, all the callbacks whose term is less than entry's term are stale.
                 self.notify_not_leader(cmd);
             }
+            // Update the leader lease when the leader commits the first empty entry
+            // during its term.
+            if self.is_leader() && term == self.raft_group.raft.term {
+                let now = clocktime::now_monotonic_raw_clocktime();
+                let next_expired_time =
```

Comment:
Why not just update `leader_lease_expire_time` to `become_leader_time + election_timeout` directly at L530? That way we don't need an extract `become_leader_time` field and don't need to guarantee all empty entries should be the first entry at their term.